lsat/store.go

```diff
@@ -22,9 +24,13 @@ type Store interface {
 	// HasToken returns true if the store contains a token.
 	HasToken() bool
 
-	// Token returns the token that is contained in the store or an error
-	// if there is none.
-	Token() (*Token, error)
+	// CurrentToken returns the token that is currently contained in the
+	// store or an error if there is none.
+	CurrentToken() (*Token, error)
+
+	// AllTokens returns all tokens that the store has knowledge of, even
+	// if they might be expired.
+	AllTokens() ([]*Token, error)
```

Comment:
Should update the `StoreToken` `godoc` comment to indicate all tokens should be kept around with in-tact date information.

Reply:
Done.